### ISSUE TITLE
ValueError: loaded state dict contains a parameter group that doesn't match the size of optimizer's group

### ISSUE THREAD

**Issue by Hdusk** [Justin-Tan/high-fidelity-generative-compression#27]
It happened when I run 'python3 train.py --model_type compression_gan --regime med --warmstart -ckpt path/to/trained/model --likelihood_type logistic' with 'HIFIC-med'

**Comment by Justin-Tan** [comment]
Yeah I think somewhere along the line I modified the models to make them more efficient and this got out of sync with the pretrained models - so you'll have to train them yourself unfortunately.
